.github/dependabot.yml

```diff
@@ -265,6 +265,10 @@ updates:
     directory: "/internal/coreinternal"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/internal/docker"
+    schedule:
+      interval: "weekly"
```

Comment:
Is this related to the same change?

Reply:
It was removed, I just run `make gendependabot`